jvm/src/main/java/com/microsoft/bond/protocol/TaggedProtocolReader.java

```diff
@@ -3,71 +3,182 @@
 import com.microsoft.bond.BondDataType;
 
 /**
- * Reads a serialized payload.
+ * Reads from serialize payload encoded using a tagged protocol.
  */
 public interface TaggedProtocolReader {
-    final class ReadFieldResult {
+
+    /**
+     * Contains the result of reading a field of a struct.
+     */
+    public final class ReadFieldResult {
```

Comment:
Removed redundant modifier

---
In reply to: [116108730](https://github.com/tstein/bond/pull/4#discussion_r116108730) [](ancestors = 116108730)